src/utils/predicate.ts

```diff
@@ -16,5 +16,5 @@ export const asTypePredicate = <T>(regularPredicate:Predicate<T>):TypePredicate<
 export const anyOf = <T>(...predicates:Array<Predicate<T>>):Predicate<T> => 
     (input:T) => predicates.reduce(
         (anyTrue, currentPredicate) => anyTrue || currentPredicate(input),
-        false
+        false as boolean
```

Comment:
wondering if this is really necessary. I don't get a type error when it's omitted

Reply:
Ooo really! I did. R u talking yarn tsc or..? Maybe something changed that made it unnecessary! :hopeful:

Reply:
Ooo really! I did. R u talking yarn tsc or..? Maybe something changed that made it unnecessary! :hopeful:

Reply:
ah nevermind!! I was just talking about syntax highlighting but `yarn tsc` does give an error

Reply:
Do we know for sure what typescript version your syntax highlighter is making use of? Could explain the inconsistency

Reply:
yeah it was 3.333, just realized my IDE wasn't using the typescript installed through yarn!